core/designsystem/src/main/java/com/crisiscleanup/core/designsystem/component/Button.kt

```diff
@@ -282,7 +283,7 @@ fun WorkTypePrimaryAction(
     onClick: () -> Unit = {},
 ) = CrisisCleanupButton(
     // TODO Common dimensions
-    modifier = Modifier.widthIn(100.dp),
+    modifier = Modifier.testTag("workTypePrimaryAction_$text").widthIn(100.dp),
```

Comment:
Normally the test tags would be constant string identifiers. The text would then be ascertained by accessing the text property. Is that no possible here?

Reply:
Nvm. I get what you're doing. You want to differentiate between multiple buttons on the same screen. Ignore the comment above.